autopush/metrics.py

```diff
@@ -7,9 +7,9 @@
 )
 
 from twisted.internet import reactor
-from txstatsd.client import StatsDClientProtocol, TwistedStatsDClient
-from txstatsd.metrics.metrics import Metrics
+from txstatsd.metrics.metrics import Metrics  # noqa
```

Comment:
nit: we can kill this and change the one import for it in `autopush/web/webpush.py` to `autopush.metrics.IMetrics`, then remove the txstatsd dep entirely